src/plugins/score-lib-state/State/Expression.cpp

```diff
@@ -27,6 +27,65 @@ bool operator<(const State::ExprData& lhs, const State::ExprData& rhs)
 {
   return lhs.toString() < rhs.toString();
 }
+
+bool findAddressInExpression(const State::Expression& expr, const State::Address& addr){
+
```

Comment:
clang-format :D
if you want you can just enable it so that it auto formats on save in qt creator